### DISCUSSION TITLE
Alarms sometimes not raised

### DISCUSSION THREAD

**Discussion by asadullah1993** [NightscoutFoundation/xDrip#2239] [Q&A] [answered]
**Hi, I'm facing an issue that sometimes my alarms are not raised. 
This occurs for both High as well as Low alerts. The images I'll be sharing are of a High Alert, which didn't raise, or was raised, but didn't rearise after snooze.**

**Any help in resolving this issue would be highly appreciated. Thanks :)**

High Value.png
![High Value](https://user-images.githubusercontent.com/55249960/180654566-7fa915c4-10f9-4e57-a611-b32ae50b3345.png)
Alert Properties.png
![Alert Properties](https://user-images.githubusercontent.com/55249960/180654574-ed96a876-3eff-4a29-ab60-d9d09bb6ea48.png)
Not Snoozed - But not working.png
![Not Snoozed - But not working](https://user-images.githubusercontent.com/55249960/180654575-60a04f58-f054-4261-a1a8-cd39fc60fb31.png)

**These are the logs in descending order of Time.**
![Log 1](https://user-images.githubusercontent.com/55249960/180654763-dc3bb92d-6968-4d13-8685-bfd8ce93ba3b.png)
![Log 2](https://user-images.githubusercontent.com/55249960/180654765-3b758e92-6cb4-4d93-b08d-f4766a0e16bf.png)
![Log 3](https://user-images.githubusercontent.com/55249960/180654771-824379b1-b9a2-4947-bf0c-364832e233b4.png)
![Log 4](https://user-images.githubusercontent.com/55249960/180654779-49854ca4-edb1-4067-8344-f41c0fd63e97.png)
![Log 5](https://user-images.githubusercontent.com/55249960/180654786-689524a3-004d-4469-a174-05451cbfd8f6.png)
![Log 6](https://user-images.githubusercontent.com/55249960/180654791-5a5fddd7-6177-4e2d-8058-c1e06be1f143.png)
![Log 7](https://user-images.githubusercontent.com/55249960/180654805-5976977a-c611-48b0-a8a9-15aee53c6592.png)
!

**Comment by Navid200** [comment, marked as answer]
Looks like you snoozed the alert till 19:34.
After 19:34, your readings have been going down.  Is it possible that you have enabled smart snoozing?
You can check under Settings &#8722;> Alarms and Alerts &#8722;> Glucose Alerts Settings

**Reply by asadullah1993** [reply]
1. Yes, snoozed till 19:34, but it didn't go off after this time as well.
2. Yes, it's enabled. Should I disable these so that the alarm goes off when the threshold is met?

**Reply by Navid200** [reply]
What do you mean it didn't go off "as well"?  
It did go off once.  That's when you snoozed it.  So, why do you say as well?

Please read the description (summary) of smart snooze.  It states that after you snooze the alert, it will stay snoozed as long as the readings are moving in the right direction.
So, if you want the alert to alert you after the snooze time is over regardless of which direction the readings are headed, you should disable the smart snooze setting.

**Reply by asadullah1993** [reply]
Yeah, you're right.
Thank you for all the help, Navid. 
Hopefully, this fixes it 😊
